src/libraries/System.Private.CoreLib/src/System/Attribute.cs

```diff
@@ -47,6 +50,8 @@ public override bool Equals(object? obj)
             return true;
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2006:UnrecognizedReflectionPattern",
+            Justification = "Unused fields don't make a difference for hashcode quality")]
```

Comment:
I assume we're OK with the hascode value itself changing as a result of trimming, right?

Reply:
I would think so. We can make this invariant if we keep all fields of attributes in linker, but I don't think we care so much.

Whoever cares about these hashcodes is probably already overriding this implementation (this returns the hashcode of the first non-null field and if that field also happens to be unused - that's a pretty bad hashcode).